pkg/controllers/job/state/completing.go

```diff
@@ -33,6 +34,7 @@ func (ps *completingState) Execute(action v1alpha1.Action) error {
 			return false
 		}
 		status.State.Phase = vcbatch.Completed
+		metrics.RegisterJobCompleted()
```

Comment:
Can we use Prometheus to test whether these metrics are indeed effective?

Reply:
What do you mean, I don't understand.

Reply:
Have we verified whether this metric is effective?

Reply:
I'm trying to test it in a local deployment environment.